### DISCUSSION TITLE
my kafka cluster wont create

### DISCUSSION THREAD

**Discussion by islet5** [strimzi/strimzi-kafka-operator#9245] [Q&A]
k8s version: azure aks v1.26.6

strimzi operator version: 0.23.0
<img width="948" alt="image" src="https://github.com/strimzi/strimzi-kafka-operator/assets/9824473/b37133e8-76a8-4ed1-bf6d-6cd1efcfb475">

[xiaowei_zhou@azure-k8s kafka]$ **cat kafka-persistent-new.yaml**
apiVersion: kafka.strimzi.io/v1beta2
kind: Kafka
metadata:
  name: kafka-sensor-new
spec:
  kafka:
    version: 2.6.0
    replicas: 3
    readinessProbe:
      initialDelaySeconds: 15
      timeoutSeconds: 5
    livenessProbe:
      initialDelaySeconds: 15
      timeoutSeconds: 5
    listeners:
      - name: plain
        port: 9092
        type: internal
        tls: false
      - name: tls
        port: 9093
        type: internal
        tls: true
    config:
      offsets.topic.replication.factor: 3
      transaction.state.log.replication.factor: 3
      transaction.state.log.min.isr: 2
      num.partitions: 3
      default.replication.factor: 3
      log.message.format.version: "2.6"
      log.retention.hours: 168
    storage:
      type: persistent-claim
      class: s-ssd-dashboard-ai-rdr
      size: 32Gi
      deleteClaim: false
  zookeeper:
    replicas: 3
    readinessProbe:
      initialDelaySeconds: 15
      timeoutSeconds: 5
    livenessProbe:
      initialDelaySeconds: 15
      timeoutSeconds: 5
    storage:
      type: persistent-claim
      size: 32Gi
      class: s-ssd-dashboard-ai-rdr
      deleteClaim: false
  entityOperator:
    topicOperator: {}
    userOperator: {}
  kafkaExporter:
    topicRegex: ".*"
    groupRegex: ".*"

error:
<img width="1435" alt="image" src="https://github.com/strimzi/strimzi-kafka-operator/assets/9824473/aee1f58b-189e-4af7-8a03-cc230097f8be">

<img width="955" alt="image" src="https://github.com/strimzi/strimzi-kafka-operator/assets/9824473/1f34cf17-7ad1-4d3f-8aa8-14a5dfd167b4">

**Comment by scholzj** [comment]
0.23 is several years old and does not support Kubernetes 1.25 and newer where you seem to run based on the error message. You will need to update to a newer Strimzi version (at least 0.29, but preferably something much more up-to-date). More details can be found here: https://strimzi.io/downloads/
